index/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"os"
+
+	"github.com/0xsequence/chainalysis"
+)
+
+func main() {
+	file, err := os.OpenFile("sanctioned_addresses.json", os.O_CREATE|os.O_RDWR, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+	chainalysis.FetchAndUpdateSanctionedAddresses(file, 16734673, 17118381)
```

Comment:
im confused about the number arguments here, what is it for..?

Reply:
its the starting and ending block, for which we fetch the logs... right now the contract has had updates from 16734673 - 17118381 only, nothing before or after that

Reply:
hmm.. but surely they will update it again sometime in the future..? having a hardcoded ending block number means we have to deploy a change, and defeats purpose of this library.